packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -435,7 +435,7 @@ const Dropdown = ({
                     />
                 </div>
                 {!error && hint && (
-                    <Text as='p' color='less-prominent' size='xxs' className='dc-dropdown__hint'>
+                    <Text as='p' color='general' size='xxs' className='dc-dropdown__hint'>
```

Comment:
Our subtexts are usually `--text-less-prominent` (gray), so please check with the design team if we should change it to black.

Reply:
checked, it is right

Reply:
Why are we changing this in the components package and not in the cashier package? 

`Components` package has **reusable** components that we **don't** change since they're being used elsewhere. Can we move this `Text` wrap to `Cashier` instead?

Reply:
Done